### ISSUE TITLE
On Staging, formatting is not updated/nice for registration pages, 4xx pages

### ISSUE THREAD

**Issue by jacobdgm** [DDMAL/CantusDB#1347]
For example,
![Screenshot 2024-03-05 at 2 43 06 PM](https://github.com/DDMAL/CantusDB/assets/58090591/f6db626f-62f7-4b8a-8526-b278a214d7e0)
![Screenshot 2024-03-05 at 2 43 25 PM](https://github.com/DDMAL/CantusDB/assets/58090591/95412554-3c03-40ab-928d-d1bc14d905e2)

I don't think this should be enough to prevent deployment to Production of the updated formatting currently on Staging. We should fix it, though.

Also, Chant Delete page

**Comment by jacobdgm** [comment]
We should also check our 5xx pages that are served by Nginx - I don't believe any of these include the header bar, but someone should at least check this.
